packages/gatsby-plugin-netlify/README.md

```diff
@@ -95,6 +95,24 @@ have to add to every path. To make this easier, the plugin provides the
 }
 ```
 
+#### Removing default headers
+
+This plugin provides default headers for increased security. If you need to remove any of those headers you can in the transfromHeaders option:
+
+```
+options: {
+    transformHeaders: (headers, path) => {
+        // Remove default security headers to allow cross-origin iframe
+        if (path === "/*") {
+            return _.pickBy(headers, (value) => {
```

Comment:
Using lodash in this example to keep this example conscience. Happy to replace with more verbose vanilla JS if needed.